packages/nimbus/src/components/toolbar/toolbar.recipe.ts

```diff
@@ -0,0 +1,54 @@
+import { defineRecipe } from "@chakra-ui/react";
+
+export const toolbarRecipe = defineRecipe({
+  className: "nimbus-toolbar",
+  base: {
+    display: "inline-flex",
+    p: "var(--toolbar-spacing)",
+    gap: "var(--toolbar-spacing)",
+    borderRadius: "300",
+    alignItems: "stretch",
+    flexDirection: "var(--toolbar-direction)",
+
+    "& .nimbus-group, & .nimbus-toggle-button-group__root": {
```

Comment:
Instead of adding new compound components due to specific styling needs, targeting the expected component's css class allows us to use the component directly with all necessary styles applied

Reply:
I'm pretty sure we do - but also, we should wait until we have all our components before going through and doing an update